fetch/fetch.go

```diff
@@ -367,6 +399,7 @@ func (f *Fetch) requestHashBatchFromPeers() {
 	var requestList []requestMessage
 	f.activeReqM.RLock()
 	for hash, req := range f.activeRequests {
+		f.log.Info("batching %v", hash.ShortString())
```

Comment:
maybe write a single log line specifying all batched hashes instead of this? maybe move this debug..